scripts/us_usda/nass_census/sv.csv

```diff
@@ -0,0 +1,159 @@
+name,sv,unit
```

Comment:
Based on the data I'm getting from the api, many of them don't have statvar mappings here (e.g. "FOOD FISH, CATFISH, WATER, IN PRODUCTION - ACRES OF WATER"). How do you propose we deal with those?

Reply:
You can see all short descriptions they support here: https://quickstats.nass.usda.gov/api/get_param_values?key=569256A0-9CF4-34F0-86F3-FC477003330A&param=short_desc

Reply:
I think ideally we add mappings for these (and add corresponding schema for them), but if we just want to refresh the existing variables, we can skip the missing ones (this list only contains a veryyyyy small subset of agriculture data )

Reply:
I'll leave that up to you and @pradh since I don't have enough context yet. For now, perhaps add a TODO to add missing mappings.

Reply:
added todo